server/swap/swap.go

```diff
@@ -293,17 +296,19 @@ type Swapper struct {
 }
 
 // NewSwapper is a constructor for a Swapper.
-func NewSwapper(cfg *Config) *Swapper {
+func NewSwapper(net dex.Network, cfg *Config) *Swapper {
```

Comment:
Please add the network as a field in the `Config`.

Reply:
I really did not envision creating functions in the dex package that do the same thing repeatedly and complicate consumers like Swapper.  I think the dex package should set and check everything as needed.  Please see my suggestion at https://github.com/decred/dcrdex/pull/432#pullrequestreview-429800118.